reckoner/chart.py

```diff
@@ -178,9 +175,6 @@ def install(self, namespace=None, context=None):
 
         Arguments:
         - namespace (string). Passed in but will be overridden by Chart().namespace if set
-
```

Comment:
I've updated documentation here because this method didn't actually return a bool value